lib/modules/commentTools.js

```diff
@@ -832,6 +832,17 @@ const magicPlaceholders = [
 			return date.toDateString();
 		},
 	}, {
+		matches: ['flair'],
+		description: 'On comments pages, returns the flair text of the original link',
+		handle() {
+			if (isPageType('comments')) { 
+				var linkflair = document.querySelector('.linkflairlabel').innerHTML;
+				if (linkflair) {
+					return `${ linkflair }`;
```

Comment:
You can simply `return linkflairlabel` here, there's no point to using template strings unless you're adding more text.